net.sourceforge.symphonic.jsymphonic.yml

```diff
@@ -0,0 +1,37 @@
+id: net.sourceforge.symphonic.jsymphonic
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=host
```

Comment:
Why is host needed?

Reply:
it needs to access usb for the sony network walkman, and the home folder for music files to import and export. it doesn't support portals.

Reply:
`host` doesn't give access to USB.

Ad for music. `xdg-music`

Reply:
Usbs are usually mounted at /run/media or /media which host gives access to

Reply:
That's not USB. That's removable drives.